### DISCUSSION TITLE
pin a message

### DISCUSSION THREAD

**Discussion by mmojadad** [chatwoot/chatwoot#8778] [Feature Requests]
### Is your feature or enhancement related to a problem? Please describe.

How to pin a message in chat

### Describe the solution you'd like

Pin a message in chat

**Comment by linear** [comment]
<p><a href="https://linear.app/chatwoot/issue/PR-868/pin-a-message">PR-868 pin a message</a></p>

**Comment by pranavrajs** [comment]
@mmojadad Can you please elaborate on the requirement here? How does a pin helps in 1:1 conversation? I can see it helpful in group chats.

**Comment by mmojadad** [comment]
In the first chat with the customer, we receive information from him, for example, information about the product order
After that, during the future chats with him, we need the operators to have this basic information in front of them, and PIN provides this purpose, as it is in Telegram messaging.
